themes/nekonight/nekonight.theme.sh

```diff
@@ -0,0 +1,42 @@
+# nekonight Bash prompt with source control management
+# Theme inspired by:
+#  - Bash_it cupcake theme
+# Demo:
+# ╭─🐱 virtualenv 🐱user at 🐱host in 🐱directory on (🐱branch {1} ↑1 ↓1 +1 •1 ⌀1 ✗)
+# ╰λ cd ~/path/to/your-directory
+
+icon_start="╭─"
+icon_user=" 🐱 \[\e[1;33m\]\u\[\e[0m\]" # Yellow-colored username
+icon_host=" at 🐱 \[\e[1;36m\]\h\[\e[0m\]" # Cyan-colored hostname
+icon_directory=" in 🐱 \[\e[1;35m\]\w\[\e[0m\]" # Magenta-colored current working directory
+icon_branch=" on (🐱 $(git_prompt_info))" # Git branch and status information
+icon_end="╰─\[\e[1m\]λ\[\e[0m\]" # Final lambda character in bold
+
+function git_prompt_info() {
+  local branch_name=$(git symbolic-ref --short HEAD 2>/dev/null)
+  local git_status=""
+
+  if [[ -n $branch_name ]]; then
+    git_status="$branch_name $(scm_git_status)"
+  fi
+
+  echo -n "$git_status"
+}
+
+function scm_git_status() {
```

Comment:
This function doesn't overwrite an existing function, yet we should use the function name of the form `_omb_theme_nekonight_*`.